src/views/Header.js

```diff
@@ -99,10 +99,22 @@ class Header extends React.Component<void, HeaderProps, void> {
     return undefined;
   }
 
+  _getHeaderTitleStyle(navigation: Navigation): ?object {
+    const header = this.props.router.getScreenConfig(navigation, 'header');
+    if (header && header.titleStyle) {
+      return header.titleStyle;
+    }
+    return undefined;
+  }
+
   _renderTitleComponent = (props: SubViewProps) => {
-    const color = this._getHeaderTintColor(props.navigation);
+    let titleStyle = this._getHeaderTitleStyle(props.navigation);
+    if (!titleStyle) {
+      const color = this._getHeaderTintColor(props.navigation);
+      titleStyle = color && ({ color });
+    }
     const title = this._getHeaderTitle(props.navigation);
-    return <HeaderTitle style={color && ({ color })}>{title}</HeaderTitle>;
+    return <HeaderTitle style={titleStyle}>{title}</HeaderTitle>;
```

Comment:
@grabbou what do you mean? the author is setting `tintColor` only if `titleStyle` is not given, above